### DISCUSSION TITLE
presentation

### DISCUSSION THREAD

**Discussion by evelynrodriguezc** [community/community#144956] [New to GitHub]
Hey there! I'm a Systems Engineer from Colombia, I'm happy to be here, I'm looking to learn more about the GitHub community

**Comment by dalvarez2596** [comment]
Eso!!! bienvenida a la comunidad :D, por lo general los comentarios nuevos de introducción la gente los agrega por acá https://github.com/orgs/community/discussions/23204, hay varios chats interesantes incluso hay uno donde muestra el [profile](https://github.com/orgs/community/discussions/121833).
Siento responder en español, pero como vi que eras parce me deje llevar :D.

**Reply by evelynrodriguezc** [reply]
no supe cómo jajaja, pero gracias!

**Comment by ghostinhershell** [comment]
Welcome to GitHub and the GitHub Community @evelynrodriguezc 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
